pkg/datasource/csvsource/read_klines.go

```diff
@@ -0,0 +1,59 @@
+package csvsource
+
+import (
+	"encoding/csv"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+// KlineReader is an interface for reading candlesticks.
+type KlineReader interface {
```

Comment:
name it `KLineReader`, we use `KLine` instead of `Kline`

Reply:
sure